src/plays/personal-profile-card/PersonalProfileCard.tsx

```diff
@@ -0,0 +1,96 @@
+import PlayHeader from 'common/playlists/PlayHeader'
+import React, { useState, ChangeEvent, useEffect } from 'react'
+import { ToastContainer, toast } from 'react-toastify'
+
+import useLocalStorage from '../../common/hooks/useLocalStorage'
+import ProfileType from './types'
+import ProfileCard from './components/profile-card'
+import ProfileForm from './components/profile-form'
+import './index.css'
+
+const initialStateProfile = {
+  name: '',
+  description: '',
+  dp: '',
+  cover: '',
+  facebook: '',
+  twitter: '',
+  instagram: '',
+  github: '',
+  linkedin: ''
+}
+
+const PersonalProfileCard = (props: any) => {
+  const [localStoreProfile, setLocalStoreProfile] = useLocalStorage("profile_card", null);
+  const [profileCard, setProfileCard] = useState<ProfileType>(initialStateProfile)
+  const [value, setValue] = useState<ProfileType>(initialStateProfile)
+  const [key, setKey] = useState<number>(0)
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement> | ChangeEvent<HTMLTextAreaElement>):void => {
+    setValue((value: ProfileType) => {
+      return { ...value, [e.target.name]: e.target.value }
+    })
+  }
+
+  useEffect(() => {
+    setProfileCard(localStoreProfile)
+    localStoreProfile !== null && setValue(localStoreProfile)
+  }, [key])
+
+  const handleClick = () => {
+    setKey(Math.random())
+    setLocalStoreProfile(value)
+  }
+
+  const handleUpload = (e: ChangeEvent<HTMLInputElement>) => {
+    const files = e.target.files[0]
+    if (files) {
+      if(files.size > 102400) {
+        toast.warn("Please upload image file size less than 100kb.");
```

Comment:
i was talking the images which are inside images folder in in this play. like cover.png is 220kb reduce it and 100kb the limit. and here user can upload any image